pkg/controllers/scheduling/schedule.go

```diff
@@ -140,6 +142,12 @@ func (s *pluginScheduler) Schedule(
 	for _, cluster := range filtered {
 		scoreSum[cluster.Name] = 0
 	}
+	// define steady prioritizer weight, default is PolicyTypeNotSteady
+	steadyWeight := 0.1
+	if placement.Spec.ChurningPolicy.PolicyType == clusterapiv1alpha1.PolicyTypeSteady {
```

Comment:
looks like ChurningPolicy is optional, shall we check its value here?

Reply:
ChurningPolicy is out of scope based on latest https://github.com/open-cluster-management-io/enhancements/pull/16/files, will remove this part.